### ISSUE TITLE
Can we port this to English in a fork

### ISSUE THREAD

**Issue by rajatkb** [7kms/react-illustration-series#91]
Just wanted to know if it's fine to fork and create an English variant for this document. Just want to extend some help to the larger community that might not be able to have access to it. If there is an English variant to it would help as well.

**Comment by 7kms** [comment]
Yes, but  the 18.x version is in processing now, 17.x is a bit outdated. 
Maybe translate the 18.x document into English is more appropriate.

**Comment by captain1023** [comment]
作者大大想冒昧的请问下什么时候可以看到18.x版本的解析呀~❤️

**Comment by 7kms** [comment]
> 作者大大想冒昧的请问下什么时候可以看到18.x版本的解析呀~❤️

不好意思，有点慢，如果8月份不能完全出输出的话，就要推迟到12月份了。
主要是我现在不做前端了，react和我的工作没有接轨的地方，完全需要单独抽时间整理。下半年还要准备考试，只能是尽量抓紧时间了。

**Comment by rajatkb** [comment]
@7kms  `Maybe translate the 18.x document into English is more appropriate.` -- Ya makes more sense. I found this to be the most latest available one hence the intent to translate this one. But ya will follow updates here for whenever version 18 is out. :) . Awesome work on this repo.

**Comment by captain1023** [comment]
> > 作者大大想冒昧的请问下什么时候可以看到18.x版本的解析呀~❤️
> 
> 不好意思，有点慢，如果8月份不能完全出输出的话，就要推迟到12月份了。 主要是我现在不做前端了，react和我的工作没有接轨的地方，完全需要单独抽时间整理。下半年还要准备考试，只能是尽量抓紧时间了。

感谢作者大大回复🙏,之前版本的解析有帮到我好多🙏🙏🙏

**Comment by ToNewDay** [comment]
> > 作者大大想冒昧的请问下什么时候可以看到18.x版本的解析呀~❤️
> 
> 不好意思，有点慢，如果8月份不能完全出输出的话，就要推迟到12月份了。 主要是我现在不做前端了，react和我的工作没有接轨的地方，完全需要单独抽时间整理。下半年还要准备考试，只能是尽量抓紧时间了。

学习总结能力太强了，请问是如何做到的
